Makefile

```diff
@@ -27,11 +27,6 @@ export PATH := $(PWD)/bin:$(PATH)
 
 all: manager
 
-# Generate docs
-.PHONY: docs
-docs:
```

Comment:
This target should remain as it is used by the updater script in `logging-operator-docs` repo.